omniport/core/base_auth/managers/get_user.py

```diff
@@ -56,20 +56,20 @@ def get_user(username):
         q_institute_webmail_address = Q(institute_webmail_address=username)
         q = (
                 q_primary_phone_number
-                | q_secondary_phone_number
                 | q_email_address
                 | q_institute_webmail_address
         )
-        contact_information = ContactInformation.objects.get(q)
-        entity = contact_information.entity
-        if type(entity) is Person:
-            user = entity.user
-            if user is not None:
-                return user
+        entity_supported = ContentType.objects.get_for_model(Person)
```

Comment:
Use a descriptive name like `person_type` as the value returned is the content **type** for the given model.